compiler/rustc_codegen_llvm/src/llvm/ffi.rs

```diff
@@ -967,161 +967,179 @@ pub type GetSymbolsErrorCallback = unsafe extern "C" fn(*const c_char) -> *mut c
 
 unsafe extern "C" {
     // Create and destroy contexts.
-    pub fn LLVMContextDispose(C: &'static mut Context);
-    pub fn LLVMGetMDKindIDInContext(C: &Context, Name: *const c_char, SLen: c_uint) -> c_uint;
+    pub(crate) fn LLVMContextDispose(C: &'static mut Context);
+    pub(crate) fn LLVMGetMDKindIDInContext(
+        C: &Context,
+        Name: *const c_char,
+        SLen: c_uint,
+    ) -> c_uint;
 
     // Create modules.
-    pub fn LLVMModuleCreateWithNameInContext(ModuleID: *const c_char, C: &Context) -> &Module;
-    pub fn LLVMGetModuleContext(M: &Module) -> &Context;
```

Comment:
I noticed that `LLVMGetModuleContext` is being completely removed here, since it became unused in #131829.

That's fine, but I'm going to split it out into a separate patch (in this PR) so that it's easier to see in history and doesn't get hidden in the mass migration to `pub(crate)`.